### DISCUSSION TITLE
occured error: in weakly_canonical: Permission denied ["store/a22/a22adc3f-6737-47cc-9732-8dcbca795931/tmp-fetch_all_2_2_0/checksums.txt"]

### DISCUSSION THREAD

**Discussion by ahuiiiiiii** [ClickHouse/ClickHouse#43961] [Q&A]
![image](https://user-images.githubusercontent.com/49714450/205683609-3ad2224b-36f6-4120-a8b2-c35b8372257f.png)
As shown in the figure，Hope to get help。Thank you very much。

**Comment by tsolodov** [comment]
Most likely it's related to permissions on file system level.

Please make sure ClickHouse process has full access to the `/var/lib/clickhouse/store/a22/a22adc3f-6737-47cc-9732-8dcbca795931/tmp-fetch_all_2_2_0/checksums.txt` file.

**Reply by tsolodov** [reply]
and all directories under `/var/lib/clickhouse/store`

**Comment by nameco11** [comment]
> ![image](https://user-images.githubusercontent.com/49714450/205683609-3ad2224b-36f6-4120-a8b2-c35b8372257f.png) As shown in the figure，Hope to get help。Thank you very much。

how to fix this ? tks
